chcemvediet/templates/customsearch/search.html

```diff
@@ -0,0 +1,18 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+{% extends "main/base/single_column.html" %}
+{% load trans from i18n %}
+
+{% comment %}
+ %
+ % Context:
+ %  -- None
+ %
+{% endcomment %}
+
+
+{% block title %}{% trans 'customsearch:search:title' %} | {{ block.super }}{% endblock %}
+
+{% block content %}
+  <script async src="https://cse.google.com/cse.js?cx=010601888809869439804:rk-fpmmkuio"></script>
+  <div class="gcse-searchresults-only"></div>
```

Comment:
Zislo by sa trochu ucesat css pre vysledky:
- Zrusit padding okolo vysledkov, nech text vysledkov je vertikalne zarovnany s nasim textom na obrazovke
- Resetnut font family a velkost fontu vo vysledkoch, aby sa inheritoval font family a velkost fontu z `body`

Sprav `chcemvediet/static/main/css/03.gcsefix.css` a importni ho v `settings/common.py` analogicky ako je `02.jqueryfix.css`. A sprav v nom prislusne css overridy. Pozor na to, ze google css sa loaduje neskor, nez nase css, takze nase overridy musia mat vyssiu specificitu.

Reply:
:ok: